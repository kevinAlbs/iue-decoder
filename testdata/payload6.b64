BnjG/o5QT0VhmxzQeAbrhfwCfGsYmaa/rfV/k1h2jkJTh7S5xtIMxFzyK0lSIZenQJCepp5KrRY3dAGup9shwz9pOUF+cAh8KXuA