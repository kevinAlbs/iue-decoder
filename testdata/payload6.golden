BlobSubtype=6 (FLE2UnindexedEncryptedValue)
KeyUUID=78c6fe8e504f45619b1cd07806eb85fc
OriginalBsonType=2
Ciphertext=7c6b1899a6bfadf57f9358768e425387b4b9c6d20cc45cf22b49522197a740909ea69e4aad16377401aea7db21c33f6939417e70087c297b80
